run-tests.sh

```diff
@@ -299,6 +310,13 @@ case "$1" in
 		exit_code=$?
 		posttest_cleanup
 	;;
+	open_api_testing)
```

Comment:
Add `open_api_testing` to the "test type" options in the usage statement please.

Reply:
Please see 43ac06f6b63a2898bd9f5090b522ddf8f3ebfaa5.